crates/aptos/src/move_tool/mod.rs

```diff
@@ -564,8 +564,14 @@ impl CliCommand<&'static str> for TestPackage {
                     self.move_options.bytecode_version,
                     self.move_options.language_version,
                 ),
-                compiler_version: self.move_options.compiler_version,
-                language_version: self.move_options.language_version,
+                compiler_version: self
```

Comment:
Why are these changes needed? This should just be the same logic, based on different defaults in MoveOptions?

Reply:
I made this change mainly to make sure that so long as move_options is not set to V1 explicitly, it will always pass V2 to the compiler config.